alpha/apps/kaltura/lib/myEntryUtils.class.php

```diff
@@ -959,7 +994,17 @@ public static function resizeEntryImage( entry $entry, $version , $width , $heig
 					$convertedImagePath = myFileConverter::convertImage($orig_image_path, $processingThumbPath, $width, $height, $type, $bgcolor, true, $quality, $src_x, $src_y, $src_w, $src_h, $density, $stripProfiles, $thumbParams, $format,$forceRotation);
 				}
 				if ($thumbCaptureByPackager && file_exists($packagerResizeFullPath))
-					unlink($packagerResizeFullPath);
+				{
+					if($bif)
+					{
+						$imgPathsToUnlink[] = $packagerResizeFullPath;
```

Comment:
Since this 2 arrays are always with same content, we can hold only one bifImgPaths , and once done with creating the BIF, delete all files linked by the array.